app/domain/authentication/handler/authentication_handler.rb

```diff
@@ -77,7 +77,7 @@ def call(parameters:, request_ip:)
       end
 
       def handle_error(err)
-        @logger.warn(@authentication_error.new(err.inspect))
+        @logger.warn(@authentication_error.new(err.message))
```

Comment:
@telday, after talking about this more with @jvanderhoof and @andytinkham, would you please delete this line?

We shouldn't be logging at the warn level here because many of these exceptions are expected and intended to be used as control flow for the application logic. So this is going to add a LOT more volume to the logs than we would like.

Jason has made a note to revisit how to add some diagnostic logging for unexpected exceptions, but this line should just be removed for 12.7.

Thanks!